openapi/oh.yaml

```diff
@@ -5848,72 +5848,6 @@ paths:
                   $ref: "#/components/schemas/EncounterDTO"
       security:
       - bearerAuth: []
-  /encounters/{code}/opds:
```

Comment:
please merge your branch with the last version of sokponta then update the oh.yaml